Applications/Terminal/Terminal.cpp

```diff
@@ -135,6 +134,36 @@ static inline Color lookup_color(unsigned color)
     return Color::from_rgb(xterm_colors[color]);
 }
 
+void Terminal::escape$l(const ParamVector& params)
+{
+    int mode = 2;
+    if (params.size() > 0) {
+        mode = params[0];
+    }
+    if (!m_question_param) {
```

Comment:
Consider passing this in as a parameter instead of using a member variable? Less error prone, and more obvious what it's for then.

Reply:
This doesn't seem to have been done, not sure why it was marked as resolved?